cranelift/native/src/lib.rs

```diff
@@ -105,6 +102,18 @@ pub fn builder_with_options(
         }
     }
 
+    if cfg!(target_arch = "aarch64") {
```

Comment:
This must use `#[cfg]` instead of `if cfg!()` as `is_aarch64_feature_detected` is not defined on non-aarch64 architectures.